src/js/game/LevelMVC/LevelView.js

```diff
@@ -1150,13 +1150,18 @@ module.exports = class LevelView {
       this.playExplosionAnimation(playerPosition, facing, destroyPosition, blockType, completionHandler, true, entity);
     };
 
-    if (LevelBlock.isFlat(blockType)) {
+    if (LevelBlock.skipsDestructionOverlay(blockType)) {
       // "flat" blocks are by definition not cube shaped and so shouldn't accept
       // the cube-shaped destroy overlay animation. In this case, destroy the
       // block immediately without waiting for the animation.
       afterDestroy();
     } else {
-      const destroyOverlay = this.actionGroup.create(-12 + 40 * destroyPosition[0], -22 + 40 * destroyPosition[1], "destroyOverlay", "destroy1");
+      let destroyOverlay = this.actionGroup.create(-12 + 40 * destroyPosition[0], -22 + 40 * destroyPosition[1], "destroyOverlay", "destroy1");
+      if (LevelBlock.isFlat(blockType)) {
+        let cropRect = new Phaser.Rectangle(0, 0, 60, 40);
```

Comment:
nit: both of these `let`s can be `const`s

Reply:
So they -can-
I was assuming that .crop() was modifying data somehow and that const would conflict with that, but that appears to -not be the case-.

Will update right now.

Reply:
`const` only limits your ability to assign a new value to the variable, not your ability to modify the already-assigned variable;